qiskit/pulse/__init__.py

```diff
@@ -13,11 +13,202 @@
+
+Let’s review what we’ve done, using :py:func:`~qiskit.pulse.Schedule.draw`:
+
+.. code:: ipython3
+
+    sched.draw(label=True)
+
+
+
+
+.. image:: ../../docs/source_images/pulse_imgs/getting_started_with_pulse_7_0.png
+
+
+
+The ways in which schedules can be composed is covered in detail in the
+:py:class:`qiskit.pulse.Schedule` documentation.
+
+This pulse will drive qubit 0. It is modulated at qubit 0’s resonant
+frequency, so it will drive the :math:`|0\rangle` to :math:`|1\rangle`
+transition. It is not calibrated to stop at a particular state, so we
+won’t know what state we’ve prepared until we look at the results. For
+our purposes, we don’t mind what state we end up in.
+
+All that’s left to do is to add a measurement. There is a convenient
+utility function for adding measurements, but it requires data from the
+backend system that the program will be running on. We will also need
+the backend to execute the program.
+
+Grab a backend
+~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    from qiskit.test.mock import FakeAlmaden
+
+    backend = FakeAlmaden()
+
+Add measurements
+~~~~~~~~~~~~~~~~
+
+Now we can use this backend to add the measurement instructions for us.
+
+.. code:: ipython3
+
+    from qiskit.scheduler.utils import measure_all
+
+    sched = sched.insert(sched.duration, measure_all(backend))
+
+Let’s see what the convenience function has added for us, using draw
+again. The acquisition and measurement pulses are very long compared to
+our initial pulse, so we can use the ``plot_range`` argument to clip the
+schedule.
+
+.. code:: ipython3
+
+    sched.draw(plot_range=[0, 1000])
+
+
+
+
+.. image:: ../../docs/source_images/pulse_imgs/getting_started_with_pulse_13_0.png
+
+
+
+There is a new pulse on :py:class:`~qiskit.pulse.channels.MeasureChannel` ``m0``, a shorthand
+name for
+``MeasureChannel(0)``. This channel stimulates readout on qubit 0.
+Likewise, ``a0`` is shorthand for ``AcquireChannel(0)``. The
+:py:class:`~qiskit.pulse.instructions.Acquire`
+instruction on ``a0`` tells the measurement devices when to begin
+collecting data on their analog-to-digital converters (ADC), and for how
+long. It is drawn as an amplitude 1 constant pulse.
+
+Execute
+~~~~~~~
+
+Our schedule is done! We can’t use our mocked backend to execute
+programs, but if you have an IBM Quantum account, you could use
+``backend = IBMQ.load_account().get_backend(open_pulse=True)`` to see if
+you have an OpenPulse enabled backend. Once you have such a backend, we
+can execute it the same way we execute circuits:
+
+::
+
+   job = execute(sched, backend)
+
+That’s it! To get the results, use ``result = job.result()``. You’ve
+created a Pulse program schedule, containing an operation on qubit 0
+followed by a measurement, executed the experiment on the backend and
+retrieved the results. A good next step in learning about Pulse would be
+to check out how to build :py:class:`qiskit.pulse.Instruction` s.
+
+
+Pulse API (:mod:`qiskit.pulse`)
```

Comment:
Was moving this to the bottom intensional? When Pulse API is clicked from the Terra API Reference, `Getting Started with OpenPulse` doesn't show up first.

Reply:
I would absolutely love a recommendation for this to show up on a different page